scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -335,9 +335,11 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
             }
 
             function validate(orig, item) {
+            	var updated = {};
```

Comment:
I fixed the indentation.
On helpers.extendItem(orig, item); is changed the orig item
There are cases when the save will not work well because after a change on an item field we will get the same value on orig field.